lib/Common/Exceptions/ReportError.cpp

```diff
@@ -106,4 +106,10 @@ __declspec(noinline) void FromDOM_NoScriptScope_fatal_error()
     ReportFatalException(NULL, E_UNEXPECTED, EnterScript_FromDOM_NoScriptScope, scenario);
 }
 
+__declspec(noinline) void Debugger_AttachDetach_fatal_error()
+{
+    int scenario = 5;
```

Comment:
It is an unused parameter which was put with the idea to group different kinds of fata error, but is not used. All other functions have this. I can put any number.